pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -66,6 +66,8 @@ in stdenv.mkDerivation ({
 
   postPatch = ''
     patchShebangs make
+    substituteInPlace erts/etc/unix/Install.src  \
+      --replace "#!/bin/sh" "${stdenv.shell}"
```

Comment:
is the `#!` dropped here intentional?

Reply:
Oh my.. Changed the match to be more precise and forgot to update the replacement.. Thank you for catching this!